pkg/api/lookout/lookout.proto

```diff
@@ -0,0 +1,38 @@
+syntax = 'proto3';
+
+package lookout;
+
+import "google/protobuf/empty.proto";
+import "google/api/annotations.proto";
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "pkg/api/queue.proto";
+
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.stringer_all) = true;
+
+message SystemOverview {
+    repeated QueueInfo queues = 1;
+}
+
+message JobInfo {
+    api.Job job = 1;
+}
+
+message QueueInfo {
+    string queue = 1;
+
+    int32 jobs_queued = 2;
```

Comment:
These could probably all be uint32, but I am not sure it's worth the effort to change.